pages/index.js

```diff
@@ -37,9 +54,21 @@ function Home(props) {
           Search for a Holiday!
         </h1>
         <h2>Countries</h2>
```

Comment:
On click of `List holidays` browser url should update to `?country=&year=` format. Also code should work on page refresh for selected year

Reply:
Targeting this for next PR. The holiday will be clickable, URL change and popup